torch/backends/__init__.py

```diff
@@ -57,6 +59,72 @@ def __getattr__(self, attr):
         return self.m.__getattribute__(attr)
 
 
+class FP32Precision:
```

Comment:
Do we actually expect this to be public API?
I don't expect so, otherwise you would need to update the doc for it: https://docs-preview.pytorch.org/pytorch/pytorch/125888/backends.html

Reply:
No, we do not need them to be public API. We renamed it to `_FP32Precision`